goodsKill-spring-boot-provider/goodsKill-service/src/test/java/org/seckill/service/test/base/BaseServiceConfigForTest.java

```diff
@@ -2,30 +2,219 @@
 
 import com.github.pagehelper.PageInfo;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.seckill.api.service.SeckillService;
+import org.seckill.entity.Seckill;
+import org.seckill.entity.SuccessKilled;
+import org.seckill.mp.dao.mapper.SeckillMapper;
+import org.seckill.mp.dao.mapper.SuccessKilledMapper;
 import org.seckill.service.GoodsKillRpcServiceSimpleApplication;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.redis.core.RedisTemplate;
 import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
 
 import javax.annotation.Resource;
+import java.util.Date;
+import java.util.Map;
+import java.util.concurrent.*;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.locks.Condition;
+import java.util.concurrent.locks.ReentrantLock;
 
 /**
  * Created by heng on 2017/6/28.
  */
 @SpringBootTest(classes = GoodsKillRpcServiceSimpleApplication.class)
 @RunWith(SpringRunner.class)
 @Ignore
+@Transactional
 public class BaseServiceConfigForTest {
     @Resource
     private SeckillService seckillService;
+    @Resource
+    private RedisTemplate redisTemplate;
+    @Resource
+    private SeckillMapper seckillMapper;
+    @Resource
+    private SuccessKilledMapper successKilledMapper;
+
+    private AtomicBoolean atomicBoolean = new AtomicBoolean(false);
+    private AtomicBoolean signal = new AtomicBoolean(false);
+    private Map<Thread, BlockingQueue> taskQueue = new ConcurrentHashMap(100);
+    private AtomicInteger updateCount = new AtomicInteger(0);
+    private ReentrantLock reentrantLock = new ReentrantLock();
+    private Condition condition = reentrantLock.newCondition();
 
     @Test
-    public void test(){
+    public void test() {
         PageInfo seckillList = seckillService.getSeckillList(0, 10, "小");
         seckillList.getList().forEach(System.out::println);
         Assert.assertTrue(seckillList.getPageSize() > 0);
     }
+
+    @Test
+    public void test1() throws InterruptedException {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/275688448/goodsKill/pullRequest?prid=5766634)